pkg/environment/environment.go

```diff
@@ -65,33 +69,24 @@ func MustGetKrewPathsFromEnvs(envs []string) KrewPaths {
 		Index:    filepath.Join(base, "index"),
 		Install:  filepath.Join(base, "store"),
 		Download: filepath.Join(os.TempDir(), "krew"),
+		Bin:      filepath.Join(base, "bin"),
 	}
 }
 
-func getKubectlPluginsPath(envs []string) string {
-	envvars := parseEnvs(envs)
-
-	// Look for "${KUBECTL_PLUGINS_PATH}"
-	if path, ok := envvars["KUBECTL_PLUGINS_PATH"]; ok && path != "" {
-		return path
-	}
-
-	// TODO(lbb): make os call testable! (dep injection)
-	// Look for "${XDG_DATA_DIRS}/kubectl/plugins"
-	if path := envvars["XDG_DATA_DIRS"]; path != "" {
-		fullPath := filepath.Join(path, "kubectl", "plugins")
-		stat, err := os.Stat(fullPath)
-		if err == nil && stat.IsDir() {
-			return fullPath
-		}
-	}
+func getKubectlPluginsPath(_ []string) string {
+	// envvars := parseEnvs(envs)
+	// // Look for "${KREW_HOMEDIR}"
+	// if path, ok := envvars["KREW_HOMEDIR"]; ok && path != "" {
+	// 	return path
+	// }
 
 	// Golang does no '~' expansion
 	return filepath.Join(homedir.HomeDir(), ".kube", "plugins")
```

Comment:
I think now that plugins don't have to be in ~/.kube/plugins, do you think we should start using ~/.krew ?

People `rm -rf ~/.kube` to reset their kubectl settings all the time. (I did that myself accidentally the other day).